pkgs/top-level/all-packages.nix

```
@@ -3971,6 +3973,8 @@ in
     mkFranzDerivation = callPackage ../applications/networking/instant-messengers/franz/generic.nix { };
   };
 
+  freac = callPackage ../applications/audio/freac { inherit boca smooth; };
```

```suggestion
  freac = callPackage ../applications/audio/freac { };
```
I don't think this is needed here but please test it.